lms/djangoapps/teams/models.py

```diff
@@ -48,9 +49,11 @@ def create(cls, name, course_id, description, topic_id=None, country=None, langu
         """
 
         team_id = generate_unique_readable_id(name, cls.objects.all(), 'team_id')
+        discussion_id = 'team_id:' + team_id
```

Comment:
This is an interesting point you bring up about needing a unique discussion ID. Now that I think about it, even this isn't necessarily a complete solution as it depends on assumed implementation details as the basis for unique discussion IDs.

I think the only way we can know with confidence that a discussion ID is unique is if we leave it up to the comments_service to create them for us.